core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -526,6 +537,19 @@ private[spark] class ExecutorAllocationManager(
       allocationManager.synchronized {
         stageIdToNumTasks(stageId) = numTasks
         allocationManager.onSchedulerBacklogged()
+
```

Comment:
Add a short comment here: "// Compute the number of tasks requested by the stage on each host"